aries_cloudagent/protocols/present_proof/v2_0/messages/pres_format.py

```diff
@@ -54,6 +54,13 @@ class Format(Enum):
                 ".formats.dif.handler.DIFPresFormatHandler"
             ),
         )
+        VCDI = FormatSpec(
+            "dif/",
```

Comment:
are we sure this should be identical to DIF above?  if they are identical then why do we have separate one?

Reply:
@supersonicwisd1 ^

Reply:
For presentation we are using the DIF Presentation Exchange from https://github.com/hyperledger/aries-rfcs/blob/main/features/0510-dif-pres-exch-attach/README.md. I was following the attachment format.